source/operations/cert-manager/cert-manager-tenants.rst

```diff
@@ -0,0 +1,252 @@
+.. _minio-certmanager-tenants:
+
+========================
+cert-manager for Tenants
+========================
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 1
+
+The following procedures create and apply the resources necessary to use cert-manager for the TLS certificates within a tenant.
+
+.. note::
+
+   The procedures use ``tenant-1`` as the name of the tenant.
+   
+   Replace the string ``tenant-1`` throughout the procedures to reflect the name of your tenant.
+
+Prerequisites
+-------------
+
+- `kustomize <https://kustomize.io/>`__ installed
+- ``kubectl`` access to your ``k8s`` cluster
+- Completed the steps to :ref:`set up cert-manager <minio-setup-certmanager>`
+- The MinIO Operator installed and :ref:`set up for cert-manager <minio-certmanager-operator>`.
+
+1) Create the tenant namespace CA Issuer
+----------------------------------------
+
+Before deploying a new tenant, create a Certificate Authority and Issuer for the tenant's namespace.
+
+1. If necessary, create the tenant's namespace.
+
+   .. code-block:: shell
+      :class: copyable
+
+      kubectl create ns tenant-1
+
+   This much match the value of the ``metadata.namespace`` field in the tenant's YAML.
+
+2. Request a Certificate for a new Certificate Authority with ``spec.isCA`` set to ``true``.
+
+   Create a file called ``tenant-1-ca-certificate.yaml`` with the following contents:
+
+   .. code-block:: yaml
+      :class: copyable
+      :emphasize-lines: 7,8
+
+      # tenant-1-ca-certificate.yaml
+      apiVersion: cert-manager.io/v1
+      kind: Certificate
+      metadata:
+        name: tenant-1-ca-certificate
+        namespace: tenant-1
+      spec:
+        isCA: true
+        commonName: tenant-1-ca
+        secretName: tenant-1-ca-tls
+        duration: 70128h # 8y
+        privateKey:
+          algorithm: ECDSA
+          size: 256
+        issuerRef:
+          name: selfsigned-root
+          kind: ClusterIssuer
+          group: cert-manager.io
+
```

Comment:
Same comment:

This must match the name of the ClusterIssuer created in the previous guide right?

We should state that after the YAML, along with a sentence to the effect of 'If you specified a different ClusterIssuer name *or* are using a different Issuer from the guide, modify the `issuerRef` to match your environment.'